### PR TITLE
CLOUD-1256: kieserver CI tests resource collisions

### PR DESCRIPTION
**Pull request by errantepiphany**
CLOUD-1256: kieserver CI tests resource collisions
https://issues.jboss.org/browse/CLOUD-1256

### CODE REVIEW

File: kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java [hunk 1]
```diff
@@ -47,7 +49,10 @@
                 @TemplateParameter(name = "KIE_SERVER_PASSWORD", value = "${kie.password:Redhat@123}")
         }
 )
-public class DecisionServerBasicMulltiContainerTest extends DecisionServerBasicTest {
+@OpenShiftResources({
+        @OpenShiftResource("https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/secrets/decisionserver-app-secret.json")
+})
+public class DecisionServerBasicMulltiContainerTest extends DecisionServerTestBase {
```
Comment (osnipso) [inline kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java]: There was a typo already in the name of this class. Maybe take the chance to fix it?

File: kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java [hunk 2]
```diff
@@ -47,7 +49,10 @@
                 @TemplateParameter(name = "KIE_SERVER_PASSWORD", value = "${kie.password:Redhat@123}")
         }
 )
-public class DecisionServerBasicMulltiContainerTest extends DecisionServerBasicTest {
+@OpenShiftResources({
+        @OpenShiftResource("https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/secrets/decisionserver-app-secret.json")
```
Comment (osnipso) [inline kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java]: Why use double indent for annotation parameters, when using single indent everywhere else? (several occurrences, this is a sample).

**Comment by osnipso** [comment]
Tested, it solves the resources issue. Some minor, cosmetic comments regarding the code. I can fix it myself, should it be the case. Should I proceed?

File: kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java (hunk 2 above)
Comment (errantepiphany) [inline kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java]: I was just following the pattern that was already in most of the files. :/

File: kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java (hunk 1 above)
Comment (errantepiphany) [inline kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java]: Ha! Didn't notice that. Yeah, we should probably fix it at some point.

**Pull request merged**